src/components/Notifications/Notifications.component.js

```diff
@@ -16,22 +19,43 @@ const Notifications = ({
   handleNotificationDismissal,
   message,
   open,
+  kind,
   showQueuedNotificationIfAny
 }) => (
-  <Snackbar
+  <Snackbar 
     {...config}
     open={open}
     ContentProps={{
       variant: 'elevation',
       'aria-describedby': 'message-id'
-    }}
-    message={<span id="message-id">{message}</span>}
-    autoHideDuration={NOTIFICATION_DELAY}
+      }}
+    message={<span id="message-id">{message}</span>}  
+    autoHideDuration={NOTIFICATION_DELAY} 
     onClose={handleNotificationDismissal}
-    // show any queued notifications after the
-    // present one transitions out
-    onExited={showQueuedNotificationIfAny}
-  />
+    // show any queued notifications after the  
+    // present one transitions outhandleNotificationDismissal
+    onExited={showQueuedNotificationIfAny}>
+    {kind==='refresh' && <MuiAlert elevation={6} variant="filled" onClose={handleNotificationDismissal} severity='info'>
+      <span id="message-id">{message}</span>
+      <Button variant="outlined">
+        refresh
```

Comment:
This should use a message.

Reply:
Thank you very much for your feedback! it's really important for me your professional advice.

Reply:
Of course! Thank you for contributing to Cboard :-)

Reply:
is a pleasure! 💯